vortex-ipc/src/message_reader.rs

```diff
@@ -246,6 +246,38 @@ impl<R: VortexRead> MessageReader<R> {
         )
     }
 
+    pub fn into_array_stream(
```

Comment:
If you want to return reader you need to be able to fully consume the stream reader and convert it into array stream